packages/harmony/src/components/layout/Paper/Paper.mdx

```diff
@@ -0,0 +1,79 @@
+import {
+  Canvas,
+  Meta,
+  Title,
+  Subtitle,
+  Description,
+  Primary,
+  Controls,
+  Story,
+  Stories
+} from '@storybook/blocks'
+import { Paper } from './Paper'
+import { Box, Flex } from '..'
```

Comment:
Let's try to resolve these to the directory they come from so we don't get circular deps